### ISSUE TITLE
Is this necessary if HTTP2 is enabled 

### ISSUE THREAD

**Issue by rossmc** [antonkril/magento-rjs-config#6]
Just wondering if this will be necessary if you're using HTTP2?
https://http2.github.io/faq/#why-is-http2-multiplexed

Any thoughts?

**Comment by Januszpl** [comment]
Yes. It helps to set some critical code to load first. By default in Magento2 you can wait longer for having interactive menu (especially on mobile). Using this you can tweak things like that.

**Comment by joshuaadickerson** [comment]
Even with HTTP/2 you will still have a lot of latency waiting for each script to load. In addition, Chrome and FF limit the number of concurrent requests. Not sure about other browsers but I know in those two you will see a notice after (I think) 30 concurrent requests.
